src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
```

Comment:
The logger is incorrectly assigned to OrderServiceImpl.class instead of Main.class. This can cause confusion when reading log files because log messages from Main will appear to come from OrderServiceImpl.